analyze.sh

```diff
@@ -14,7 +14,7 @@ if [ -d "$ARTIFACT_DIR" ]; then
   cp public/dist/report.html "${ARTIFACT_DIR}"
 fi
 
-MAX_BYTES=4823450 # ~4.6 MiB
+MAX_BYTES=5700000 # ~5.4 MiB
```

Comment:
The limit increased for @patternfly/quickstarts@5.4.5 + marked dependency

The upgrade from 5.4.0 → 5.4.5 introduced the marked dependency, but removed showdown. But in the codebase we are still using showdown, therefore both packages are kept, which leads to the increase in the size of the bundle.